_includes/manuals/1.12/5.8.3_secret_token.md

```diff
@@ -0,0 +1,8 @@
+
+The last file required to make a Foreman cluster work is __$app_root/tmp/secret_token.rb__. This file should contain a secret token that is modeled after __$app_root/config/initializers/local\_secret\_token.rb__. __$app_root/tmp/secret_token.rb__ is used by __$app_root/config/initializers/secret_token.rb__ to sign cookies. Once you have created the new secret_token.rb, restart Foreman and clear Foreman's cache:
```

Comment:
I think that point #3 disproves 5.8.3 - if it's not reading `tmp/secret_token.rb` because of the extension, then it's reading the local_secret_token that I had set the same across the board. Let me re-do my testing since I can replicate this behavior in my dev environment now, it could be that I made two changes at once and attributed success to the second change.